Cargo.toml

```diff
@@ -63,4 +63,4 @@ members = [
 ]
 
 [patch.crates-io]
-diesel = { git = "http://github.com/diesel-rs/diesel", rev = "793de72" }
+diesel = { git = "http://github.com/wattsense/diesel", "branch" = "optimize_owned_row"}
```

Comment:
I will need to change this back to the diesel main repo after merging the PR there.

(That's a note for myself, I will take care of this)

Reply:
```suggestion
diesel = { git = "http://github.com/diesel-rs/diesel", "rev" = "f2eb9b2"}
```